R-packages/evalcast/R/plot_calibration.R

```diff
@@ -98,3 +55,108 @@ setup_wedgeplot <- function(calib) {
                                   prop_above = "Proportion above",
                                   prop_below = "Proportion below"))
 }
+
+
+#' ggplot formatting for wedge calibration plot.
+#'
+#' @param calib the output from [compute_calibration()]
+#' @param grp_vars variables over which to compare calibration. These
+#'   determines the color of the lines and faceting depending on `type`
+#' @param facet_rows A variable name to facet data over. Creates a
+#'   separate row of plots for each value of specified variable. Can be used
+#'   with `facet_cols` to create a grid of plots.  Should be passed to 
+#'   plot_calibration when customized.
+#' @param facet_cols Same as `facet_rows`, but with columns.
+#'
+#' @return ggplot object
+format_wedgeplot <- function(calib, 
+                             grp_vars,
+                             facet_rows = "forecast_date",
+                             facet_cols = "forecaster"){
+  if (!is.null(facet_rows)) {
+    non_grouped_facet <- setdiff(facet_rows, grp_vars)
+    assert_that(length(non_grouped_facet) == 0,
+                msg = paste("Variables must be grouped in order to be faceted in rows:",
+                            non_grouped_facet))
+  }
+  if (!is.null(facet_cols)) {
+    non_grouped_facet <- setdiff(facet_cols, grp_vars)
+    assert_that(length(non_grouped_facet) == 0,
+                msg = paste("Variables must be grouped in order to be faceted in cols:",
+                            non_grouped_facet))
+  }
+  
+  facet_layer <- facet_grid(rows = vars(!!!syms(facet_rows)),
```

Comment:
Do we want this to always be a grid instead of `facet_wrap`?

Reply:
I like facet_grid so that you can sort visually across different categories, is there a reason to use facet_wrap instead?

Reply:
If you facet on only one category, then wrap works better.

Reply:
I think this is fine. Did the `...` get fixed?

Reply:
The documentation or removing it?

Reply:
replacing it with `facet_rows` and `facet_cols` explicitly.

Reply:
Done.  If they're both null they just use the defaults for each plot type.